src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/ServerCrudTests.ps1

```diff
@@ -85,29 +85,34 @@ function Test-GetServer
 {
 	# Setup
 	$rg = Create-ResourceGroupForTest
+	$rg1 = Create-ResourceGroupForTest
 	$server1 = Create-ServerForTest $rg
 	$server2 = Create-ServerForTest $rg
+	$server3 = Create-ServerForTest $rg1
 
 	try
 	{
 		# Test using parameters
 		$resp1 = Get-AzureRmSqlServer -ResourceGroupName $rg.ResourceGroupName -ServerName $server1.ServerName
 		Assert-AreEqual $server1.ServerName $resp1.ServerName
-		Assert-AreEqual $server1.ServerVersion $resp1.ServerVersion
 		Assert-AreEqual $server1.SqlAdministratorLogin $resp1.SqlAdministratorLogin
 		
 		# Test piping
 		$resp2 = $server2 | Get-AzureRmSqlServer
 		Assert-AreEqual $server2.ServerName $resp2.ServerName
-		Assert-AreEqual $server2.ServerVersion $resp2.ServerVersion
 		Assert-AreEqual $server2.SqlAdministratorLogin $resp2.SqlAdministratorLogin
-
+		
 		$all = Get-AzureRmSqlServer -ResourceGroupName $rg.ResourceGroupName
-		Assert-AreEqual $all.Count 2
+		Assert-AreEqual 2 $all.Count
+
+		# Test getting all servers in all resource groups
+		$all2 = Get-AzureRmSqlServer
+		Assert-AreEqual 3 $all2.Count
```

Comment:
This will fail if I run it in my subscription that already has some sql servers unrelated to PS test. Can you change this so that it checks that the list contains the 3 previous servers?